server/config/passport.ts

```diff
@@ -27,10 +27,19 @@ passportUse(new LocalStrategy({
         .select('username email hashed_password salt')
         .exec((err, user) => {
             if (err) return done(err);
+            let errorObject = { message: 'wrong username or password' };
             if (!user || !user.authenticate(password)) {
-                return done(null, false, { message: 'wrong username or password' });
+                return done(null, false, errorObject);
```

Comment:
Nothing important, actually. It’s just like you have one function (`done`) which behaves in a different way (for instance, does not take the same amount of arguments) according to its second argument (which can be either an object or a boolean from my understanding). I come from a strongly-typed language world d:.

Nonetheless, I think you are ready to merge.